scipy/linalg/_basic.py

```diff
@@ -68,10 +68,14 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
         Square input data
     b : (N, NRHS) array_like
         Input data for the right hand side.
-    sym_pos : bool, optional
-        Assume `a` is symmetric and positive definite. This key is deprecated
-        and assume_a = 'pos' keyword is recommended instead. The functionality
-        is the same. It will be removed in the future.
+    sym_pos : bool, optional, deprecated
+        Assume `a` is symmetric and positive definite.
+
+        .. deprecated:: 0.19.0
+           This key is deprecated and ``assume_a = 'pos'`` keyword is
+           recommended instead. The functionality is the same. `sym_pos`
+           will be removed in SciPy 1.11.0.
```

Comment:
IMO the sentence is quite bumpy (it was already before so not your fault), but still:
```suggestion
        .. deprecated:: 0.19.0
           This keyword is deprecated and should be replaced by using
           `assume_a = 'pos'`. `sym_pos` will be removed in SciPy 1.11.0.
```